app/src/main/res/layout/activity_main.xml

```diff
@@ -4,14 +4,14 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="#3D5AF8"
     tools:context=".MainActivity">
 
-    <com.facebook.login.widget.LoginButton
-        android:id="@+id/login_button"
+    <androidx.fragment.app.FragmentContainerView
```

Comment:
Please describe your idea. You wanna show here 2 fragments on the same page?

Reply:
Firstly, I combined all fragments on a one page, but then I divided them for different pages via Navigation component